src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  height: 408px;
```

Comment:
It is a bad idea to hardcode the height because the height should depend on the content inside the card.

Reply:
fixed